pkg/bmc/ilo5.go

```diff
@@ -0,0 +1,47 @@
+package bmc
+
+import (
+	"net/url"
+)
+
+type iLO5AccessDetails struct {
+	hostname string
+	path     string
+}
+
+const iLO5DefaultScheme = "https"
+
+func init() {
+	registerFactory("ilo5+virtualmedia", newiLO5VirtualMediaAccessDetails)
+}
+
+func newiLO5VirtualMediaAccessDetails(parsedURL *url.URL) (AccessDetails, error) {
+	return &iLO5AccessDetails{
+		hostname: parsedURL.Hostname(),
+		path:     parsedURL.Path,
+	}, nil
+}
+
+func (a *iLO5AccessDetails) Type() string {
+	return "ilo5"
```

Comment:
This method is used for debugging, so for consistency with other BMC plugins it should return the full URL scheme from the parsedURL.